ansible_3par_docker_plugin/container/container.sh

```diff
@@ -0,0 +1,5 @@
+docker rmi -f hpestorage/legacyvolumeplugininstaller:3.1
+docker rmi -f container
+docker image build --build-arg http_proxy=http://web-proxy.in.hpecorp.net:8080 --build-arg https_proxy=http://web-proxy.in.hpecorp.net:8080 --no-cache -t container .
```

Comment:
1. Rename `container.sh` to something to specific to show it as a build script.
2. Parameterize the proxy settings via $1 , $2